src/category_theory/limits/has_limits.lean

```diff
@@ -491,8 +491,7 @@ lemma has_limits_of_size_shrink [has_limits_of_size.{(max v₁ v₂) (max u₁ u
 ⟨λ J hJ, by exactI has_limits_of_shape_of_equivalence
   (ulift_hom_ulift_category.equiv.{v₂ u₂} J).symm⟩
 
-@[priority 100]
-instance has_smallest_limits_of_has_limits [has_limits C] :
+lemma has_smallest_limits_of_has_limits [has_limits C] :
```

Comment:
This makes the `simp_nf` linter time out.
Because if the morphisms of `C` are plain Types, then `has_limits C` is by definition `has_limits_of_size.{0 0} C`